src/routes/logs.ts

```diff
@@ -0,0 +1,15 @@
+
+import { Router } from 'express';
+
+const router = Router();
+
+router.get('/paths', (req, res) => {
+  res.json({
+    paths: [
+      '/data/recording/lte-status.log',
+      '/data/recording/odc-api.log',
```

Comment:
i think we are missing one here map-ai. Can you check with @derekhive or @punov

Reply:
I think that's everything, please note that `/data/recording/hivemapper-data-logger.log` and `/data/recording/redis-handler.log` are not present yet

Reply:
Yes, `/data/recording/map-ai.log` needs to be there!

Reply:
also @mazar-farran just pass the file name like "odc-api.log" because I call it from /public/odc-api.log